engines/rancher_on_aks/app/controllers/rancher_on_aks/wrapup_controller.rb

```diff
@@ -7,8 +7,9 @@ def show
 
       @fqdn = RancherOnAks::Fqdn.load()
       @password = Helm::Rancher.last.initial_password
-      @resource_group = Azure::ResourceGroup.first
       @cluster = Azure::Cluster.last
+      @resource_group = @cluster.resource_group
```

Comment:
what's the difference regarding the resource group ?

Reply:
There are 2 resource groups in the database: the resource group of the DNS zone, and the resource group of the cluster. This ensures we are referencing the proper resource group in the wrapup page.